package-lock.json

```diff
@@ -1507,6 +1507,7 @@
       "version": "1.1.0",
       "resolved": "https://registry.npmjs.org/ansi-colors/-/ansi-colors-1.1.0.tgz",
       "integrity": "sha512-SFKX67auSNoVR38N3L+nvsPjOE0bybKTYbkf5tRvushrAPQ9V75huw0ZxBkKVeRU9kqH3d6HA4xTckbwZ4ixmA==",
+      "dev": true,
```

Comment:
210 dependencies have had `"dev": true` added to them. Is this correct?

Reply:
package-lock.json got overriden when I did npm install. I have now reverted it as it's more sensible since no changes inside the package.json have been made